client/codegen.yml

```diff
@@ -13,3 +13,5 @@ generates:
       withHOC: false
       withHooks: true
       enumsAsTypes: true
+      scalars:
+        DateTime: String
```

Comment:
@Akirtovskis it should actually be `string` lowercase to represent the typescript type :)